libcodechecker/orm_model.py

```diff
@@ -287,6 +266,16 @@ def __init__(self, bug_hash, author, message, created_at):
         self.created_at = created_at
 
 
+class ReviewStatus(Base):
```

Comment:
How do we display these? Shouldn't status changes be shown like regular comments?